pkg/healthautoexport/types.go

```diff
@@ -455,7 +485,14 @@ func (t *Time) UnmarshalJSON(data []byte) error {
 	if string(data) == "null" {
 		return t.Time.UnmarshalJSON(data)
 	}
-	var err error
-	t.Time, err = time.Parse(`"`+TimeFormat+`"`, string(data))
-	return err
+	var s string
+	if err := json.Unmarshal(data, &s); err != nil {
+		return err
+	}
+	parsed, err := ParseTime(s)
+	if err != nil {
+		return errors.Wrapf(err, "cannot parse timestamp: %v", s)
+	}
```

Comment:
`ParseTime` already returns the unparsed string as part of the error message, I don't think we need to add it here.